### PR TITLE
adding V2 compatibility

### PR DESCRIPTION
**Pull request by worldspawn00**
adding support for new layout options

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by worldspawn00** [comment]
checks will fail till all 3 modified files are approved

**Comment by worldspawn00** [comment]
Pull 10312, 10311, 10310

**Review by ridingqwerty — APPROVED** [review]

File: keyboards/wsk/sl40/sl40.h [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
 
 #define LAYOUT( \
-    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, \
-    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111,       K113, \
-    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, \
-    K300, K301, K302,                   K306,                   K310, K311, K312, K313  \
+	K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, \
+	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111,       K113, \
+	K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, \
+	K300, K301, K302,       K304,       K306,       K308, K309, K310, K311, K312, K313  \
```
Comment (drashna) [inline keyboards/wsk/sl40/sl40.h:9]:
```suggestion
    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, \
    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111,       K113, \
    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, \
    K300, K301, K302,       K304,       K306,       K308, K309, K310, K311, K312, K313  \
```

File: keyboards/wsk/sl40/sl40.h [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 ) { \
-    { K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013 }, \
-    { K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  KC_NO, K113 }, \
-    { K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213 }, \
-    { K300,  K301,  K302,  KC_NO, KC_NO, KC_NO, K306,  KC_NO, KC_NO, KC_NO, K310,  K311,  K312,  K313 }  \
+	{ K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013 }, \
+	{ K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  KC_NO, K113 }, \
+	{ K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213 }, \
+	{ K300,  K301,  K302,  KC_NO, K304,  KC_NO, K306,  KC_NO, K308,  K309,  K310,  K311,  K312,  K313 }  \
```
Comment (drashna) [inline keyboards/wsk/sl40/sl40.h:14]:
```suggestion
    { K000,  K001,  K002,  K003,  K004,  K005,  K006,  K007,  K008,  K009,  K010,  K011,  K012,  K013 }, \
    { K100,  K101,  K102,  K103,  K104,  K105,  K106,  K107,  K108,  K109,  K110,  K111,  KC_NO, K113 }, \
    { K200,  KC_NO, K202,  K203,  K204,  K205,  K206,  K207,  K208,  K209,  K210,  K211,  K212,  K213 }, \
    { K300,  K301,  K302,  KC_NO, K304,  KC_NO, K306,  KC_NO, K308,  K309,  K310,  K311,  K312,  K313 }  \
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by noroadsleft** [comment]
Closing per https://github.com/qmk/qmk_firmware/pull/10311#issuecomment-696504518.

**Pull request closed without merging**